libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/CycleMapView.java

```diff
@@ -1,6 +1,5 @@
 package net.cyclestreets.views;
 
-import net.cyclestreets.liveride.PebbleNotifier;

```

Comment:
Confirmed - this class had a private instance field of type `PebbleNotifier` that was never used, so I cleaned it up.